resources/zib/zib-TNMTumorClassification.PnPerineuralInvasion.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-TNMTumorClassification.PnPerineuralInvasion" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-TNMTumorClassification.Pn_PerineuralInvasion" />
```

Comment:
In the profiling guidelines it is stated that the `.id` coincides with the latter part of the `.url`, but that is not the case here.

Reply:
Deleted underscore because resource.id can't contain this symbol.